core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -1255,7 +1255,7 @@ private boolean isNullable() {
                     return true;
                 }
             }
-            return false;
+            return KotlinUtil.isMarkedNullable(typeElement);
```

Comment:
Could we update the warn for redundant uses when `@Nullable`, `?` or `Optional` are used together?
https://github.com/line/armeria/blob/fe8688490187890b4fbeeb8a1579b55572bcbf3c/core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java#L1136

Reply:
>  replace @Nullable with ?(Kotlin nullable type)

I prefer a more explicit style. :-)